src/test/java/io/orangebeard/listener/OrangebeardExtensionTest.java

```diff
@@ -47,6 +48,31 @@ public void before_all_test() {
         verify(orangebeardClient).startTestItem(eq(null), any(StartTestItem.class));
     }
 
+    @Test
+    public void before_all_with_multiple_suites() {
+        // Test how the "beforeAll" method behaves if it needs to start multiple suites.
+        UUID testRunUUID = UUID.fromString("49e7186d-e14d-4eeb-bc29-e36279d3b628");
+
+        UUID uuid1 = UUID.fromString("27bf84ed-6269-4629-863d-0899078f8196");
```

Comment:
maybe use a bit more convenient names, like suite, subsuite and subsubsuite uuid? or something along those lines? Makes the test a bit more readable.